File: tests/benchmarks/cases/map_object_access/dotnet/benchmark_case.cs
namespace BenchmarkRunner;

internal static class MapObjectAccessCase
{
    public const string Name = "map_object_access";
    public const string PassBanner = "BENCH_MAP_OBJECT_ACCESS_PASS";

    public static long Run(int scale) => BenchmarkSupport.MapObjectAccess(scale);
}

File: tests/benchmarks/cases/map_object_access/java/benchmark_case.java
final class MapObjectAccessCase {
    static final String NAME = "map_object_access";
    static final String PASS_BANNER = "BENCH_MAP_OBJECT_ACCESS_PASS";

    private MapObjectAccessCase() {}

    static long run(int scale) {
        return BenchmarkSupport.mapObjectAccess(scale);
    }
}
